### PR TITLE
drafting human task orchestration guide

### PR DESCRIPTION
**Pull request by christinaausley**
Going ahead and adding guide into GH/Markdown for future edits and suggestions so we can merge when ready.

### CODE REVIEW

File: docs/guides/getting-started-orchestrate-human-tasks.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+---
+id: getting-started-orchestrate-human-tasks
+title: Getting started with Human Task Orchestration
+sidebar_label: Getting started with Human Task Orchestration
+description: "Efficiently allocate work through user tasks."
+keywords: [human tasks, orchestration, getting-started]
```
Comment (akeller) [inline docs/guides/getting-started-orchestrate-human-tasks.md:6]:
```suggestion
keywords: [human tasks, orchestration, getting-started, user task]
```
I haven't had time to look this up or investigate, but do you know if keywords can be two words or do they need to be dashed? I've been dashing two word keywords, but I don't know if that's required. 🤷‍♀️

File: docs/guides/getting-started-orchestrate-human-tasks.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+### Create an automated process with user tasks
+
+To create an automated process with user tasks, take the following steps:
+
+#### Develop your automated process with user tasks
+
+1. Log in to your Camunda Cloud account.
+2. To create a BPMN diagram, navigate to Web Modeler via the **Modeler** tab, and click **New project**.
+3. Name your project and select **New > BPMN Diagram > + Create blank**.
```
Comment (akeller) [inline docs/guides/getting-started-orchestrate-human-tasks.md:34]: I like this streamlined approach!

File: docs/guides/getting-started-orchestrate-human-tasks.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+4. Give your model a descriptive name, and then give your model a descriptive id within the **General** tab inside the properties panel on the right side of the screen.
```
Comment (akeller) [inline docs/guides/getting-started-orchestrate-human-tasks.md:35]: Can you include an example of a descriptive name and id? 

Also, I noticed for Web Modeler, the name of the diagram (as in the file) does NOT update when you update the name and id on the properties panel. I didn't call this out in the MO guide updates, but can you check with @ultraschuppi and team if this is expected behavior? For completeness, we should really encourage naming the diagram file.

File: docs/guides/getting-started-orchestrate-human-tasks.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+In our example process, we’ll take a look at two user tasks: choosing what to eat for dinner, and eating dinner.
+
+![example process human tasks](./img/human-task-1.png)
+
+#### Start and view your process instance
+
+1. To deploy your diagram to your cluster, click **Deploy diagram > Deploy**.
+2. To officially start a new process instance, click **Start instance**.
+3. Navigate to [Operate](../components/operate/index.md) by clicking the honeycomb icon next to the **Start instance** button.
+4. Click **View process instances** to see your process instance alongside the green token waiting at the task.
```
Comment (akeller) [inline docs/guides/getting-started-orchestrate-human-tasks.md:53]:
```suggestion
4. Click **View process instances** to see your process instance alongside the green token waiting at the user task.
```

**Review by akeller — COMMENTED** [review]
@christinaausley this is looking great so far, thanks for pulling this together.

Since the BPMN diagram is not included, what I'm missing is instructions for designing the BPMN. You have the info for dragging the first user task and configuring the properties panel, but I'm missing the step where you repeat the process for the second user task (for example, do you follow it exactly and again configure like the first one?).

Since this guide is intended for someone new to Camunda and new to BPMN (or process orchestration in general) we need to remind them that all symbols involved in the flow need to be connected with an arrow. Also, remind them to include an end event! Then they can move on to starting and viewing their process instance. (I just updated some of this on the MO guide because I missed it myself in the first pass!)

File: docs/guides/getting-started-orchestrate-human-tasks.md (hunk 1 above)
Comment (christinaausley) [reply docs/guides/getting-started-orchestrate-human-tasks.md:6]: I don't think you need dashes, but I'll follow up with confirmation.

File: docs/guides/getting-started-orchestrate-human-tasks.md (hunk 4 above)
Comment (christinaausley) [reply docs/guides/getting-started-orchestrate-human-tasks.md:53]: Adjusted in latest commit.

File: docs/guides/getting-started-orchestrate-human-tasks.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+#### Develop your automated process with user tasks
+
+1. Log in to your Camunda Cloud account.
+2. To create a BPMN diagram, navigate to Web Modeler via the **Modeler** tab, and click **New project**.
+3. Name your project and select **New > BPMN Diagram > + Create blank**.
+4. Give your model a descriptive name, and then give your model a descriptive id within the **General** tab inside the properties panel on the right side of the screen.
+5. Use Web Modeler to design a BPMN flow with user tasks. Create a user task by dragging the task icon from the palette, or click the existing start event and the displayed task icon.
+6. Change the task type by clicking the wrench icon. Select **User Task**.
+7. Add a descriptive name using the properties panel.
```
[the remark this answers is not in the document]
Comment (christinaausley) [reply docs/guides/getting-started-orchestrate-human-tasks.md]: Adjusted in latest commit.

**Comment by christinaausley** [comment]
@akeller Following up on your two comments above, but have since updated the example diagram. Let me know your thoughts and/or if you'd like to make tweaks.

File: docs/guides/getting-started-orchestrate-human-tasks.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+id: getting-started-orchestrate-human-tasks
+title: Getting started with Human Task Orchestration
+sidebar_label: Getting started with Human Task Orchestration
+description: "Efficiently allocate work through user tasks."
+keywords: [human tasks, orchestration, getting started, user guide]
+---
+
+## Getting started with user task orchestration
+
+### Purpose
```
Comment (akeller) [inline docs/guides/getting-started-orchestrate-human-tasks.md:11]: You can go ahead and remove these two headers. The title is redundant with the frontmatter title value so it's duplicating and the purpose section doesn't need a title, just have it flow right into the text.

**Comment by akeller** [comment]
Love the diagram! I think it's the right amount of simplicity and introduction to various elements without being too simple or too exhaustive. I left one comment about the front matter and top two headers, but I think from my side this looks great.

Also just reminded me I still haven't put badge labels on the MO guide 🙄  do as I say not as I do 😆

File: docs/guides/getting-started-orchestrate-human-tasks.md (hunk 6 above)
Comment (christinaausley) [reply docs/guides/getting-started-orchestrate-human-tasks.md]: Resolved in latest commit!

**Comment by christinaausley** [comment]
@akeller Merging as we can follow up on the naming of the diagram file and open up a new issue if necessary once @ultraschuppi is back from vacation.

**Pull request merged**